bindings/pyroot/pythonizations/python/ROOT/pythonization/_ttree.py

```diff
@@ -65,6 +65,9 @@ def _TTreeAsMatrix(self, columns=None, exclude=None, dtype="double", return_labe
         array(, labels): Numpy array(, labels of columns)
     """
 
+    import warnings
+    warnings.warn("TTree.AsMatrix is deprecated since v6.24 and will be removed in v6.26. Please use instead RDataFrame.AsNumpy.", DeprecationWarning)
```

Comment:
Thanks! Can you use `FutureWarning`?

https://docs.python.org/3/library/exceptions.html#FutureWarning

Reply:
Thanks for pointing this out! Indeed, `FutureWarning` is the more correct :)

Reply:
Aaaand fixed!